contents/blog/dev-marketing-paid-ads.md

```diff
@@ -0,0 +1,154 @@
+
+This means taking paid ads seriously (because of the $$$), but don't spend all your time here. At PostHog, paid ads take up ~10% of our marketing team's time in an average week. We spend 80%+ on writing. 
+
+### All paid ads are basically the same
+
+Very crudely, your ads will have one of two objectives - _conversion_ or _awareness_. 
+
+Conversion means that you want people to click on your ad and then do something like sign up for your product. You can define conversion how you like - this is just a typical example. Conversion is easy to measure, which is why marketers like it. However, you are generally only _capturing_ demand from people already aware of your product. 
+
+Awareness means you want people to see your ad. Maybe they won't click on it - that's ok. Your objective is to just get the word out so that someone is exposed to your product enough times that at some point they Google 'super cool product' and come sign up. Awareness-based ads are a small part of _creating_ demand. This takes more effort to measure, but is totally possible - see below.  
+
+We split our budget 50-50 between the two. This is not a 'rule' - just don't spend it all on conversion. 
+
+### Paid ads don't work on developers
+
+Everyone who works at a tech company basically thinks they are immune to ads because they are super good at internetting, and have never seen an ad for a new product, clicked on it, and then signed up. Your _quantitative_ attribution data in PostHog or whatever analytics tool you are using will seem to back this up. 
+
+This is why we ask all users where they heard about PostHog whenever they sign up or book a demo - it's a simple (optional) free text field. Enough of our users say 'ad on Google' or similar that we know paid ads do actually reach a large chunk of them. Especially ones with money to spend on behalf of their company. However, you need to take the time to collect and digest this _qualitative_ attribution data in the first place. 
+
+### An agency will solve all your problems
+
+I'm a big fan of hiring an agency if you're a startup - paying $5-10k/mo for a small, outsourced team is way more efficient at this stage than hiring one paid ads specialist. Look in Europe - US agencies can be 2-3x more expensive. We use [Hey](https://www.heydigital.co/) and like them. 
+
+_However,_ I really recommend you get personally familiar with each of the main channels I've listed below first. Working with an agency isn't just set and forget - you need to constantly give feedback, ask questions, and suggest new ideas if you want to get the most out of them. 
+
+I personally like [the Demand Curve program](https://www.demandcurve.com/growth-program) – it helped get us off the ground and means we're much better equipped to give our agency feedback. Outsource the legwork, not your critical thinking. 
```

Comment:
It's a bit unclear what the Demand Curve program is? Should clarify it is a course teaching you about customer acquisition and growth channels.